### ISSUE TITLE
Samples in the bin-directory use JDBCFeeder, but should probably use JDBCImporter now

### ISSUE THREAD

**Issue by centic9** [jprante/elasticsearch-jdbc#633]
See e.g. https://github.com/jprante/elasticsearch-jdbc/blob/master/bin/mysql-metawiki.sh, it tries to use JDBCFeeder, which is not available any more, this should likely be changed to JDBCImporter.

See also http://stackoverflow.com/questions/32062195/elasticsearch-jdbc-initial-setup-classnotfoundexception-jdbcfeeder where this is also discussed.

**Comment by jprante** [comment]
Yes, there was a mistake when I merged branches. The files with JDBCFeeder should be ignored.
